common/display/kmsfencehandler.cpp

```diff
@@ -82,6 +90,11 @@ void KMSFenceEventHandler::HandleRoutine() {
     close(kms_fence_);
     kms_fence_ = -1;
   }
+
+  for (NativeSurface *surface : surfaces_) {
+     surface->SetInUse(false);
```

Comment:
Agree but how to make sure the display is not holding the buffer anyway need to rely on the fence only right. Maybe we can add the buffer ref count and reuse the buffer accordingly.